blockchain/v1/reactor_fsm_test.go

```diff
@@ -769,7 +769,7 @@ forLoop:
 	for i := 0; i < int(numBlocks); i++ {
 
 		// Add the makeRequestEv step periodically.
-		if i%int(maxRequestsPerPeer) == 0 {
+		if maxRequestsPerPeer == 0 {
```

Comment:
what happened to `i%`?

```suggestion
		if i%maxRequestsPerPeer == 0 {
```